### PR TITLE
NMS-8702: View current Site outages does not filter for "building"

### PR DESCRIPTION
**Pull request by agalue**
View current Site outages does not filter for "building"

* JIRA: http://issues.opennms.org/browse/NMS-8702
* Bamboo: http://bamboo.internal.opennms.com:8085/browse/OPENNMS-ONMS948

Note: the solution should be similar for Meridian 2015, but I'll cherry-pick it manually after merging this PR.

### CODE REVIEW

**Comment by agalue** [comment]
This is ready to be merged. Objections ?

File: opennms-webapp/src/main/java/org/opennms/web/outage/filter/AssetFilter.java [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+package org.opennms.web.outage.filter;
+
+import org.hibernate.criterion.Criterion;
+import org.hibernate.criterion.Restrictions;
+import org.hibernate.type.StringType;
+import org.opennms.web.filter.EqualsFilter;
+import org.opennms.web.filter.SQLType;
+
+/**
+ * The Class BuildingFilter.
```
Comment (soleger) [inline opennms-webapp/src/main/java/org/opennms/web/outage/filter/AssetFilter.java]: It still says BuildingFilter in a couple of places in this class.

Comment (agalue) [inline opennms-webapp/src/main/java/org/opennms/web/outage/filter/AssetFilter.java]: > On Aug 31, 2016, at 11:10 AM, Seth Leger <notifications@github.com> wrote:
> 
> It still says BuildingFilter in a couple of places in this class.


Upsss .... Fixed :)

**Comment by agalue** [comment]
Typos has been fixed. How does it look now ?

**Pull request merged**